server/src/AppBundle/Submission/SubmissionManager.php

```diff
@@ -104,6 +104,13 @@ public function processFor($sourceKey, RequestPayload $payload)
             $result->throwException();
         }
 
+        // Send email notifications.
+        $this->notificationManager->sendNotificationFor($submission);
+        $this->notificationManager->notifySubmission($submission, $payload->getNotify());
+
+        // Return the response.
+        return $this->callHookFor($sourceKey, 'createResponseFor', [$submission]);
```

Comment:
You cannot return here. It will prevent all subsequent code from executing.

Reply:
Good call... Oops